Move prover returns: exiting with boogie verification errors
error:  A postcondition might not hold on this return path.

    ┌── tests/sources/references.move:62:9 ───
    │
 62 │         aborts_if true;
    │         ^^^^^^^^^^^^^^^
    │
    =     at tests/sources/references.move:54:5: mut_ref_incorrect (entry)
    =     at tests/sources/references.move:55:13: mut_ref_incorrect
    =     at tests/sources/references.move:56:31: mut_ref_incorrect
    =         b = <redacted>,
    =         b_ref = <redacted>
    =     at tests/sources/references.move:39:5: mut_b (entry)
    =     at tests/sources/references.move:40:9: mut_b
    =         b = <redacted>,
    =         b = <redacted>
    =     at tests/sources/references.move:57:9: mut_ref_incorrect
    =         b_ref = <redacted>
    =     at tests/sources/references.move:58:14: mut_ref_incorrect
    =         b = <redacted>
    =     at tests/sources/references.move:59:13: mut_ref_incorrect
    =     at tests/sources/references.move:54:5: mut_ref_incorrect (exit)
